src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -354,12 +356,12 @@ namespace OpenMS
     }
 
     QByteArray querybytes;
-    querybytes.append("--" + boundary + "--\n");
+    querybytes.append(("--" + boundary + "--\n").toUtf8());
```

Comment:
Surprised that this works but I guess this means there is an implicit construction of a QString from the char* string

Reply:
I think it is more that the QString boundary has special + operators for char that create a new QString.

Reply:
Right, I think the free function:
const QString operator+(const char *s1, const QString &s2)
gets called.